Education/CodingStandards.md

```diff
@@ -183,26 +195,32 @@ This will alow you to reference said template anywhere within the page.  Any add
 ### 4.3. 
 
 ## 5. Custom Actions
-Because Custom Actions can contain code/scripts it creates a large amount of locations that a potentially buggy script could be hiding. For this reason, if your code will contain more than 5 lines, it is best to house your code in a global script, and pass it a reference to the quote, as necessary.
+Because Custom Actions can contain code/scripts it creates a large amount of locations that a potentially buggy script could be hiding. For this reason, if your code will contain more than a single line, it is best to house your code in a global script, and pass it a reference to the quote, as necessary.
 
 ```Python
 ScriptExecutor.ExecuteGlobal('GlobalScriptName', context.Quote)
 ```
 
-## 6. Tags
+## 6. Products
+Because Products can contain code/scripts it creates a large amount of locations that a potentially buggy script could be hiding. For this reason, if your code will contain more than a single line, it is best to house your code in a global script, and pass it a reference to the Product, as necessary.
+
+```Python
+ScriptExecutor.ExecuteGlobal('GlobalScriptName', Product)
```

Comment:
Maybe show the Param obj in global script like you showed me

Reply:
We actually do show that down in global scripts. I can add a link.